lib/streamlit/state/session_state.py

```diff
@@ -452,26 +452,6 @@ def __delitem__(self, key: str) -> None:
         if widget_id in self._old_state:
             del self._old_state[widget_id]
 
-    def __getattr__(self, key: str) -> Any:
-        try:
-            return self[key]
-        except KeyError:
-            raise AttributeError(_missing_attr_error_message(key))
-
-    def __setattr__(self, key: str, value: Any) -> None:
-        # Setting internal attributes must be done using the base method to
-        # avoid recursion.
-        if key in INTERNAL_STATE_ATTRS:
```

Comment:
can `INTERNAL_STATE_ATTRS` (along with the usage of the variable in `__delitem__`) now be removed?